common/src/main/java/dev/latvian/mods/kubejs/item/ItemBuilder.java

```diff
@@ -249,4 +279,39 @@ public ItemBuilder modifyAttribute(ResourceLocation attribute, String identifier
 		attributes.put(attribute, new AttributeModifier(new UUID(identifier.hashCode(), identifier.hashCode()), identifier, d, operation));
 		return this;
 	}
+
+	public interface ItemColorJS {
+		Object getColor(ItemStackJS itemStack, int tintIndex);
```

Comment:
Could this use the Rhino `Color` type as a result, or does Rhino not type wrap the *result* of a method automatically?